etp-db/src/main/sql/migration/v3-energiatodistus/v3.1-energiatodistus.sql

```diff
@@ -2,8 +2,33 @@
 create table energiatodistus (
   id int generated by default as identity primary key,
   versio int not null,
+  tila_id int not null default 0 references energiatodistustila (id),
   allekirjoituksessaaika timestamp,
   allekirjoitusaika timestamp,
   laatija_id int not null references laatija (id),
   data jsonb
 );
+
+create table energiatodistustila_event (
```

Comment:
Tätä taulua voisi mahdollisesti laajetaa myös kattamaan esim. editoinnit ja liitetiedostojen lisäämiset, sekä muut muutokset joissa energiatodistus muuttuu, mutta tila ei. Voi olla kyllä eri taulukin, jos data ei istu samaan tauluun.

Reply:
ajattelin laittaa muut muutokset eri tauluun niin tilahistoriaa on helpompi tulkita

Reply:
historia taulujahan voisi olla periaatteessa mille tahansa et-taulun tietojen alijoukolle

Reply:
noita nimeämiskäytäntöjä voisi vielä tämän osalta ehkä vähän miettiä. varsinkin jos näitä historiatauluja tulee kaikille käsitteille

Reply:
:+1: